extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -16,7 +16,16 @@
 
 import {isLayoutSizeDefined} from '../../../src/layout';
 import {user} from '../../../src/log';
+import {
+    installVideoManagerForDoc,
+} from '../../../src/service/video-manager-impl';
+import {VideoEvents} from '../../../src/video-interface';
+import {videoManagerForDoc} from '../../../src/services';
+import {assertAbsoluteHttpOrHttpsUrl} from '../../../src/url';
 
+/**
+ * @implements {../../../src/video-interface.VideoInterface}
```

Comment:
Thanks. @DarXector For now let's comment out `brid-player.amp.html` in `test-example-validation` and I will add it back when validator is in production.